code/model/modelread.cpp

```diff
@@ -3521,60 +3579,154 @@ void submodel_stepped_rotate(model_subsystem *psub, submodel_instance_info *sii)
 	}
 }
 
-void world_find_real_model_point(vec3d *out, vec3d *world_pt, polymodel *pm, int submodel_num, matrix *orient, vec3d *pos);
+void ez_debug(int index, vec3d *v1, vec3d *v2)
```

Comment:
This looks like debugging code so I'm going to assume that it will be removed for the final version of the code.